.gitignore

```diff
@@ -1,10 +1,9 @@
-.idea/*
-venv/*
-source/.idea/*
-source/__pycache__/*
-out/tensorflow_pipeline/logs/*
+__pycache__/
```

Comment:
maybe this should be **/__pycache__/ so that it ignores any pycache sub-directory

Reply:
I believe `__pycache__/` has the exact same function here as `**/__pycache__/`. All my `__pycache__` folders were ignored in the commit.